app-text/ghostwriter/ghostwriter-1.7.3.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="A cross-platform, aesthetic, distraction-free Markdown editor"
+DESCRIPTION="ghostwriter is a text editor for Markdown."
+HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
+COPYRIGHT="wereturtle et al."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/wereturtle/ghostwriter/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="34eb3855c5cf7c541128eb740ee41de4000c91f71e9a2cf464c35d77dd523c79"
+ADDITIONAL_FILES="ghostwriter.rdef.in"
```

Comment:
You forgot to add the rdef.in file.

Reply:
Its in https://github.com/kenmays/haikuports/blob/ghostwriter-1.7.3/app-text/ghostwriter/additional-files/ghostwriter.rdef.in

Reply:
It needs to be in this PR.

Reply:
Ok, i see, it is already in the tree: https://github.com/haikuports/haikuports/tree/master/app-text/ghostwriter/additional-files

Reply:
Then remove the old recipe.